CSharp/Library/Microsoft.Bot.Builder/Dialogs/PromptDialog.cs

```diff
@@ -637,7 +637,7 @@ protected override bool TryParse(IMessageActivity message, out double result)
         [Serializable]
         public class PromptChoice<T> : Prompt<T, T>
         {
-            private IDictionary<T, IEnumerable<T>> choices;
+            private IDictionary<T, IEnumerable<T>> synonyms;
```

Comment:
I think this is done https://github.com/Microsoft/BotBuilder/pull/2609/commits/2ce721f9c41b36116e7073f1a09c393d3a29426f